### PR TITLE
Add enter text step

### PR DESCRIPTION
**Pull request by uarlouski**

### CODE REVIEW

File: vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/InputActions.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     * <br>
+     * The atomic actions performed are:
+     * <ol>
+     * <li>input text into the element</li>
+     * <li>hide keyboard</li>
+     * </ol>
+     * @param element element to input text, must not be {@code null}
+     * @param text text to input into the element, must not be {@code null}
+     */
+    public void inputText(WebElement element, String text)
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/mobileapp/action/InputActions.java:44]: `typeText`

File: vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java [hunk 2]
```diff
@@ -71,8 +74,30 @@ public void tapByLocator(Locator locator)
         findElementToTap(locator).ifPresent(tapActions::tap);
     }
 
+    /**
+     * Enter <b>text</b> into the <b>element</b>
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java:78]: `Enter` -> `Type`

File: vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java [hunk 3]
```diff
@@ -71,8 +74,30 @@ public void tapByLocator(Locator locator)
...
+     * <br>
+     * The atomic actions performed are:
+     * <ol>
+     * <li>enter text into the element</li>
+     * <li>hide keyboard</li>
+     * </ol>
+     * @param text text to enter into the element
+     * @param locator locator to find an element
+     */
+    @When("I enter `$text` in field located `$locator`")
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java:88]: `enter` -> `type`

File: vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java [hunk 4]
```diff
@@ -71,8 +74,30 @@ public void tapByLocator(Locator locator)
...
+        findElement("The element to enter text", locator).ifPresent(e -> inputActions.inputText(e, text));
+    }
+
     private Optional<WebElement> findElementToTap(Locator locator)
     {
-        return baseValidations.assertElementExists("The element to tap", locator);
+        return findElement("The element to tap", locator);
+    }
+
+    private Optional<WebElement> findElement(String description, Locator locator)
```
Comment (valfirst) [inline vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java:99]: what's the profit?

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java (hunk 4 above)
Comment (uarlouski) [reply vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ActionSteps.java]: discussed to remove

**Review by ikalinin1 — APPROVED** [review]

**Review by valfirst — APPROVED** [review]

**Pull request merged**